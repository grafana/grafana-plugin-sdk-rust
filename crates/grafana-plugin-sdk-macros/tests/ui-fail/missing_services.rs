#![allow(dead_code, unused_variables)]

mod a {
    #[derive(Clone)]
    struct MyPlugin;

    #[grafana_plugin_sdk::main]
    async fn plugin() -> MyPlugin {
        MyPlugin
    }
}

fn main() {}
