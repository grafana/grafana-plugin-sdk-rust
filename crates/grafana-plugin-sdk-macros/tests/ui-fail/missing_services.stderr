error: unexpected end of input, unexpected token in nested attribute, expected ident
 --> tests/ui-fail/missing_services.rs:7:5
  |
7 |     #[grafana_plugin_sdk::main]
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `grafana_plugin_sdk::main` (in Nightly builds, run with -Z macro-backtrace for more info)
